### ISSUE TITLE
--loglevel doesnt work in 'pnpm install' for latest version 5.16.0

### ISSUE THREAD

**Issue by davidabap** [pnpm/pnpm#3122]
### Code to reproduce the issue:
I add loglevel=error in .npmrc and create a package.json only contain popper.js. compare the result between pnpm and npm
![image](https://user-images.githubusercontent.com/25033055/106872726-96ac2800-670e-11eb-8f29-65762da4f88e.png)
Just npm/pnpm install the same package.json

### Expected behavior:
WARN message should be suppress like npm
### Actual behavior:
--loglevel doenst work in .npmrc and CLI

### Additional information:

 - `node -v` prints: 10.22.0
 - Windows, macOS, or Linux?:macOS

**Comment by mcmxcdev** [comment]
This works as expected now, see:

```
$ pnpm i uuid@3 --loglevel error

dependencies:
+ uuid 3

```

**Comment by zkochan** [comment]
although `loglevel error` might mean that only errors should be printed. So not even info messsages.

**Comment by Herr-Sepp** [comment]
The title should probably be adjusted.
As zkochan said it would be great if at loglevel _warn_ or _error_ all the information about the installed packages are suppressed.

Currently (pnpm 8.9) this can not be prevented (if you dont want to use _--reporter=silent_ to supress all messages).

**Comment by jfirebaugh** [comment]
Is `--loglevel` just not working at all for `pnpm install`? When I run `pnpm install --loglevel=debug`, I don't get any additional logging beyond the default.

**Comment by Aeolun** [comment]
> Currently (pnpm 8.9) this can not be prevented (if you dont want to use --reporter=silent to supress all messages).

Yeah. I want no output unless there is a failure, in which case I want to see the error message, and only that.
